x-pack/test/functional/services/uptime.ts

```diff
@@ -8,12 +8,39 @@ import { KibanaFunctionalTestDefaultProviders } from '../../types/providers';
 
 export const UptimeProvider = ({ getService }: KibanaFunctionalTestDefaultProviders) => {
   const testSubjects = getService('testSubjects');
-
+  const wait = (seconds: number) => new Promise(r => setTimeout(r, seconds * 1000));
   return {
     async assertExists(key: string) {
       if (!(await testSubjects.exists(key))) {
         throw new Error(`Couldn't find expected element with key "${key}".`);
       }
     },
+    async navigateToPlugin() {
+      await testSubjects.click('homeSynopsisLinkuptime');
+    },
+    async setStartOfDatePickerRange(dateRangeStart: string) {
+      await testSubjects.click('superDatePickerShowDatesButton');
+      await wait(1);
+      await testSubjects.click('superDatePickerstartDatePopoverButton');
+      await wait(1);
+      await testSubjects.click('superDatePickerAbsoluteTab');
+      await wait(1);
+      await testSubjects.setValue('superDatePickerAbsoluteDateInput', dateRangeStart);
+    },
+    async dateRangeIsAlreadySet() {
+      return await testSubjects.exists('superDatePickerstartDatePopoverButton');
+    },
+    async monitorIdExists(key: string) {
+      await wait(1);
+      await testSubjects.existOrFail(key);
+    },
+    async navigateToMonitorWithId(monitorId: string) {
+      await wait(1);
+      await testSubjects.click(`monitor-page-link-${monitorId}`);
+    },
```

Comment:
I pulled this PR and I'm running it.  It fails for me here;
```
       │ debg TestSubjects.exists(monitor-page-link-auto-http-0X131221E73F825974)
       │ debg existsByDisplayedByCssSelector [data-test-subj~="monitor-page-link-auto-http-0X131221E73F825974"]
       │ info Taking screenshot "C:\Users\LeeDr\git\kibana\x-pack\test\functional\screenshots\failure\Uptime app overview page loads and displays uptime data based on date range.png"
       │ info Current URL is: http://localhost:5620/app/uptime#/?_g=()
       │ info Browser output is:
       │ info Saving page source to: C:\Users\LeeDr\git\kibana\x-pack\test\functional\failure_debug\html\Uptime app overview page loads and displays uptime data based on date range.html
       └- × fail: "Uptime app overview page loads and displays uptime data based on date range"
       │      Error: expected testSubject(monitor-page-link-auto-http-0X131221E73F825974) to exist
       │       at TestSubjects.existOrFail (C:/Users/LeeDr/git/kibana/test/functional/services/test_subjects.js:54:15)
       │       at process._tickCallback (internal/process/next_tick.js:68:7)
 ```

And I don't see a data-test-subj on the element here;
![image](https://user-images.githubusercontent.com/13542669/52238857-a9c35d80-2892-11e9-94fe-b4ed35d7e8c9.png)

Reply:
Strangely it worked for me the last time I ran it locally. I'll revisit it when I get a second.